src/main/pages/user-management/organizations.adoc

```diff
@@ -11,49 +11,66 @@ folder: user-management
 [id="organizations-in-eclipse-che"]
 == Organizations in Eclipse Che
 
-Organizations allow admins to group Eclipse Che users and allocate resources. Resources and permissions are controlled and allocated within the admin dashboard by a system administrator.
+Organizations allow administrators to group Eclipse Che users and allocate resources. The system adminstrator controls and allocates resources and permissions within the administrator dashboard.
 
 [id="roles"]
 == Roles
 
-There are two different roles in an organization:
+The following are roles in an organization:
 
-* *Admin*: Organization admins are able to manage the organization. Admins can edit settings, manage members, resources and sub-organization. +
-* *Members*: Organization members are able create workspaces, manage their own workspaces and use any workspaces they have permissions for.
-
-There is also a *System Admin* role which sits above the organization construct. System Admins are able to create root organizations, as well as manage all resources, members and sub-organizations.
+* *Administrator*
+    Manages the organization, sub-organization, members, resources, and edits settings. +
+* *Member*
+    Uses, creates, and manages workspaces.
+* *System Adminstrator*: 
+    Creates root organizations, manages resources, members and sub-organizations. System administrators have more permissions than the administrator and member. 
 
 [id="organization-structure"]
-== Organization Structure
+== Root organizations and sub-organizations 
```

Comment:
When changing a section title, the respective ID should also be changed correspondingly. However, beware that all xrefs or other links to that ID need to updated, too.

Reply:
I plan to change the respective IDs and xrefs, wanted to get feedback first on the proposed changes.

Reply:
Makes sense. Thx.